extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -354,6 +353,9 @@ public function getPage() {
 				} else {
 					$data['details'] = MercuryApiMainPageHandler::getMainPageMockedDetails( $title );
 				}
+
+				// On Curated Main Pages page title and Wiki name are equal and to avoid duplicates we don't use page title
+				$documentTitle = '';
```

Comment:
After https://github.com/Wikia/app/pull/10110/files#diff-551276c093245208cefecd540a00ba55R334 it's already an empty string.